apps/dapp/src/constants/env/types.ts

```diff
@@ -3,16 +3,54 @@ interface Subgraphs {
   protocolMetrics: string;
 }
 
-interface Contracts {}
+interface Contracts {
```

Comment:
Given we have a lovely mono repo @nichosystem - I think this should only live in one place.

I suggest deleting the list of active contracts from the protocol repo, and keeping this as the single source of truth, for active contracts WDYT?

I can leave the markdown of deprecated/deleted contracts in the protocol repo (that feels like the right home for it), but no need to keep 'active' contracts listed in 2 places

Reply:
Update, started a draft PR for this: https://github.com/TempleDAO/temple/pull/446

I'm not going to actually unify the contracts in this PR, Leave it to whoever does issue https://github.com/TempleDAO/temple/issues/435